src/flashbar/__integ__/collapsible.test.ts

```diff
@@ -30,6 +35,7 @@ describe('Collapsible Flashbar', () => {
         setupTest(async page => {
           await page.toggleCollapsibleFeature();
           await page.addInfoFlash();
+          await wait(FOCUS_THROTTLE_DELAY);
```

Comment:
If it's for saving time, it wouldn't work unfortunately because we need to wait exactly this amount of time, because of the focus throttling. The test also a bit more exact this way. What if the expected state is satisfied too early, but one the throttle delay passes it's in an undesired state? If using `waitFor`, the test would succeed but it should actually fail.